bin/zopen-audit

```diff
@@ -81,16 +97,30 @@ downloadCVEJSONCache()
     printError "Failed to obtain vulnerability json from ${JSON_VULNERABILITIES_URL}; ${curlout}"
   fi
   chtag -tc 819 "${JSON_CVE_CACHE}"
+
+  if $upgrade; then
+    LATEST_RELEASES_CACHE="${cachedir}/zopen_releases_latest.json"
```

Comment:
Minor: Wonder if you can reuse or call downloadJSONCache() here: https://github.com/ZOSOpenTools/meta/blob/main/include/common.sh#L1167
It checks if the currently downloaded json is the latest and if not, then it downloads it
(this could be done in another PR)

Reply:
Maybe we could update zopen_vulnerability.json to include a timestamp as well, and add a json file name argument to the downloadJSONCache function so that we can use it for all three json caches (releases, latest releases, vulnerabilities)?

Reply:
We're tapping into the `ETag` header response here: https://github.com/ZOSOpenTools/meta/blob/b57de6ea09e1f6d1f548b412984b25ae03815aa4/include/common.sh#L1167

We could also extend that common function to download all 3

Reply:
We can handle this in another PR, will merge what you have